clang/tools/driver/cc1_main.cpp

```diff
@@ -204,6 +238,10 @@ int cc1_main(ArrayRef<const char *> Argv, const char *Argv0, void *MainAddr) {
   if (Clang->getFrontendOpts().PrintSupportedExtensions)
     return PrintSupportedExtensions(Clang->getTargetOpts().Triple);
 
+  // --print-enabled-extensions takes priority over the actual compilation.
```

Comment:
What happens if I ask for `--print-supported-extensions` and `--print-enabled-extensions`?

I'd guess either both happen, or the last one in wins.

Reply:
Only one of them wins and is executed, depending on the order of their implementation in `cc1_main.cpp`. The same applies when using the two existing `--print-supported-extensions` and `--print-supported-cpus` together.
Ideally, we should make these three options mutually exclusive in the future.